app/views/patients/update_failed.js.erb

```diff
@@ -0,0 +1,9 @@
+// flash failure
+if ($( "#flash" ).length == 0) {
+  $('main > .container').prepend("<div id='flash'></div>")
+}
+$('#flash').removeClass().hide().fadeIn('slow');
+$('#flash').addClass("alert alert-danger")
+$('#flash').html(
+  '<button type="button" class="close" data-dismiss="alert" aria-hidden="true">×</button><div id="flash_notice"><%= @errors %></div>'
```

Comment:
Same deal, can probably render the partial here. Note that you might have it flash something in the controller action itself, see user creation for something similar I think.